### PR TITLE
Headerfooter

### PR DESCRIPTION
**Pull request by ianlovell**

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ const mergeLastContacted = sessions => {
       .then(lastContacted => sessions);
 }
 
-const getSessionsFromServer = _ => {
+const getSessionsFromServer = () => {
+    return SessionsApi.getAllSessions()
```
Comment (lclarksl) [inline app/actions/index.js]: Duplicate line

**Comment by lclarksl** [comment]
Other than duplicate line, :shipit:

File: app/components/NavLink.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    render() {
+        const defaultStyles = {
+            margin: 8
+        };
+
+        const styles = Object.assign({}, defaultStyles, this.props.styles);
+
+        return (
+      <RaisedButton label={this.props.label} onClick={e => handleClick} style={styles} />
+    )
```
Comment (csscottlogic) [inline app/components/NavLink.jsx]: Looks odd with ) and } vertically aligned?

**Comment by csscottlogic** [comment]
Apart from minor comments, :shipit:
Feel free to merge whenever you like @ianlovell

**Pull request merged**